internal/services/aws_reservation_service.go

```diff
@@ -41,7 +41,13 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if payload.Region == "" {
+		logger.Warn().Msg("No region passed to the reservation endpoint using us-east now, but this will be failure in the future.")
```

Comment:
I would not do in-code commitments, 10 years later we could still find this ;-)

    logger.Warn().Msg("No region passed, using us-east-1")

Also: We don't do full stops in logs.

Finally, I would suggest to move this logic into the client function including the warning, so all possible call behavior is the same.

Reply:
Done :)